src/main/java/org/jenkinsci/tracey/RabbitMQTrigger.java

```diff
@@ -42,13 +42,11 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
-//import org.jenkinsci.tracey.filter.EiffelEventTypeFilter.EiffelEventTypeFilterDescriptor;
-
 public class RabbitMQTrigger extends Trigger<Job<?,?>> {
 
     private static final Logger LOG = Logger.getLogger(RabbitMQTrigger.class.getName());
     private String exchange = RabbitMQDefaults.EXCHANGE_NAME;
```

Comment:
Could you please rename exchange to exchangeName? I understand that is a bit unrelated but it would be really nice to unify naming and I don't want to create a separate issue for that